docs/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/apache/flink
+
+go 1.18
+
+require (
+	github.com/apache/flink-connector-elasticsearch v0.0.0-20220406114754-ce4058113b37 // indirect
+	github.com/apache/flink-connector-elasticsearch/docs v0.0.0-20220425092437-8e4b91cb3984 // indirect
```

Comment:
does it really make to commit such a go.mod file? aren't the date/sha references gonna change all the time?

Reply:
From my experience, the go.mod reference is static. The `hugo mod get -u github.com/apache/flink-connector-elasticsearch/docs` comment is the one that actually checks if there's a new version, which would be included in a `go.sum` file. That one is not checked in.

Reply:
I think we can actually live without this. When the documentation is built, it will be added by Hugo to this file, but it's not a prerequisite. If it's not there, it will just pull the latest, which is actually what we want.

Reply:
We can't add a file to gitignore to ignore further modifications; it only ensures that a new file can't be tracked.
So it's either we have no go.mod file at all, or we have the full thing.
Maybe we could generate it on the fly during the docs build process? 🤔

Reply:
That was a good idea. I'm now creating a default `go.mod` file in the steps where it's necessary and I've added these files to the .gitignore

Reply:
Please make sure to also add the .gitignore entries to the other release branches after merging.